src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IArgument.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
Is there an intentional change on this line?

Reply:
> Is there an intentional change on this line?

VS is changing it without by involvement. I believe it is related to the fact that the tool, that IOperation team is using to automatically update baselines, is messing with BOMs. I am not using the tool and instead editing files manually in VS.